scripts/e2e-test.sh

```diff
@@ -42,6 +42,22 @@ helm template deploy/helm/eunomia-operator/ \
   --set eunomia.operator.deployment.enabled= \
   --set eunomia.operator.namespace=$TEST_NAMESPACE | kubectl apply -f -
 
+helm template deploy/helm/eunomia-operator/ \
+  --set eunomia.operator.image.tag=dev \
+  --set eunomia.operator.namespace=$TEST_NAMESPACE | kubectl apply -f -
+
+sleep 30
+
+podname=$(kubectl get pods  -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}' -n $TEST_NAMESPACE)
+
+if kubectl exec "${podname}" date -n $TEST_NAMESPACE
+   then
+     echo "Pod is Healthy"
+       else
+     echo "Pod is not Healthy"
+       exit 1
+fi
+
```

Comment:
Please move this entire block to the end of the script. I missed this detail in my first review. Sorry.

Also please add the below `helm` command at the very end ...

```
helm template deploy/helm/eunomia-operator/ \
  --set eunomia.operator.image.tag=dev \
  --set eunomia.operator.namespace=$TEST_NAMESPACE | kubectl delete -f -
```

Reply:
@seanmalloy Ok i can add `helm` very end . But i have to cleanup(delete)  namespace  again . 
Can we just add After `operator-sdk test local ...` command ?